app/helpers/application_helper.rb

```diff
@@ -9,4 +9,16 @@ def alert_class_for(flash_type)
     }
     types[flash_type.to_sym]
   end
+
+  def all_months_select_options(first_month, last_month)
+    all_months(first_month, last_month).map { |month| [month.strftime('%Y年%m月'), month] }
+  end
+
+  def all_months(first_month, last_month)
+    diff = Month(last_month) - Month(first_month)
+    return [first_month] if diff == 0
```

Comment:
__[rubocop]__
* Use `diff.zero?` instead of `diff == 0`. [:ref](https://github.com/bbatsov/ruby-style-guide#predicate-methods)

<!-- Comment by [SideCI](https://sideci.com) -->